go/runtime/bundle/registry.go

```diff
@@ -152,6 +152,37 @@ func (r *Registry) AddManifest(manifest *Manifest, dir string) error {
 	return nil
 }
 
+// Manifests returns all known manifests (regular and detached).
+func (r *Registry) Manifests() []ExplodedManifest {
```

Comment:
We have now 2 functions, `Manifests` and `GetManifests`. Not the best. I think that we will be able to remove the latter once we merge PR for upgrading ROFL components. Then, we will also be able to remove `regularManifests`.